### ISSUE TITLE
Beautiful Linux

### ISSUE THREAD

**Issue by edermania** [Enzo-zsh/Gmac-linux#1]
I don't understand why this project is stopped, it's 10 to 0 compared to many distros out there... :'( I would really like to use it. I downloaded it and installed it but I had some problems that prevented me from using it. At one point the internet connection crashes and doesn't access anymore. I can only restart it, but it crashes when restarting if there is no internet. Then you have to force it. I couldn't download Chrome and other apps :'( I wanted to use it as my main system.
Much prettier than Elementary, Ubuntu, Mint and others out there...
congratulations

**Comment by enzo-zsh** [comment]
Yes, this project was really impressive, so currently you can use pearOS, it's a macOS like Linux distros at https://pearos.xyz

(I'm not the developer of this project)
